### PR TITLE
Add docker-compose.yml for framework

### PR DESCRIPTION
**Pull request by kuba--**
Signed-off-by: kuba-- <kuba@sourced.tech>

### CODE REVIEW

**Comment by erizocosmico** [comment]
Putting this on hold until @smola makes a decision about using docker compose.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+version: '2'
```
Comment (mcarmonaa) [inline docker-compose.yml:1]: Is there a reason to not use the version 3?

Comment (kuba--) [reply docker-compose.yml:1]: @mcarmonaa - there is no any reason! I just sticked with *v2*  for a long time, so kept it here for backwards compatibility, that's it. Frankly, I'm totally open to upgrade it to any version.

**Comment by smola** [comment]
This repository is now deprecated. So not merging this.

**Pull request closed without merging**